src/Http/Middleware/IdentifyRequest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Str;
+
+class IdentifyRequest
```

Comment:
I am not sure what's the purpose of this class?

Reply:
Oh, okay. You took very literally description of task #19. What I meant was provide a way to enable any middleware for testing purposes.

I don't think we should add classes like this to testing package.